deeplearning4j/deeplearning4j-ui-parent/deeplearning4j-vertx/src/test/java/org/deeplearning4j/ui/TestVertxUIMultiSession.java

```diff
@@ -54,19 +57,25 @@
  */
 @Ignore
 public class TestVertxUIMultiSession extends BaseDL4JTest {
+    private static final Logger log = LoggerFactory.getLogger(TestVertxUIMultiSession.class);
```

Comment:
Not a big deal - but we can use Lombok `@Slf4j` annotation on the class to get the same thing

Reply:
Now using Lombok `@Slf4j` annotation.